DECISION-MAKING.md

```diff
@@ -1,143 +1,154 @@
     * Auth0 & Auth0 Lock & Auth0 Dashboard: @jdow @akrug @gdestuynder
+       * <Question: Is `Auth0 Lock` the same as `NLX`>
     * Service Integration (CIS): @akrug
     * LDAP: @jdow
     * DuoSecurity: @gdestuynder
-    * Mozillians.org: @hmitsch
-    * WorkDay: HRIS
-
-## Architects
-
-* @gdestuynder (IAM, Security)
-* @mvk-mozilla (IT)
-
-## Program Managers
-
-* @hmitsch
-
-## User Experience (UX)
-
-* @mbranson
+    
+* IT Enterprise Architecture
+   * @mvk-mozilla
+   
+* Outside-IT Capability Ownership
+    * User Experience (UX): @mbranson
+    * Mozillians.org: ParSys
+    * WorkDay (HRIS): People Team
 
-# Modules owners
+# Modules Owners
 
-This document list which team or persons are responsible for which component (module) of the Mozilla IAM stack. This is
-a more detailed model.
+This document list which team or people are responsible for which component (module) of the _IAM Stack_. This is mainly useful for decision making and is inspired by [Mozilla's Module System](https://wiki.mozilla.org/Modules).
 
-This is mainly useful for decision making and is loosely following the [Firefox Module
-system](https://wiki.mozilla.org/Modules).
+_Note:_ We don't yet know what a conflict resolution process would look like if there was a diverging opinion between the Module Owner and the IT Capability Owner for that module.
 
-- **Owner**: A single person which is accountable for the module and normally the final decision maker. In case there is
-  serious disagreement, the Service owner, Product owner, or Capability owner can still be consulted. The Capability
-  owner has final say on all decisions.
-- **Peers**: List of persons which can act on behalf of the Owner, in case the owner is unavailable.
-- **Operator**: A single person or team which is responsible for keeping the system running, generally also the
+Extensions to the Mozilla Module System:
+- **IT Capability Owner**: The person which is capability owner for a particular module.
+- **Operator**: A person or team which is responsible for keeping the system running, generally also the
   developer in a devops model.
 
 ## CIS (Change Integration Service)
 
 Backend which deals with IAM data integration from multiple sources.
 
-- Owner (accountable, decision maker): @akrug
+- IT Capability Owner: @jeffbryner
+- Module Owner: @akrug
 - Peers: @gdestuynder
-- Operator (responsible, developer): EIS
+- Operator: EIS
 
 Repositories:
 - https://github.com/mozilla-iam/cis
 - https://github.com/mozilla-iam/cis_functions
 
 ## Person API
 
-API which presents and allow manipulation of CIS data.
-
-- Owner (accountable, decision maker): @akrug
-- Peers: @gdestuynder
-- Operator (responsible, developer): EIS
+API which presents and allows manipulation of CIS data. This includes a publicly available endpoint to the _IAM Stack_ (replacing the Mozillians.org API).
 
+- IT Capability Owner: @jeffbryner
+- Owner: @akrug
+- Peers: @gdestuynder <can we add a ParSys Peer?>
```

Comment:
yes, if they really become peer - ie participate to development and have understanding of how it works

peers are not just "people with interest in the service", but people who can behave has owner if the owner is not present. otherwise, everyone else is welcome to add their input/feedback/etc. The owner makes decisions based on all feedback and data (not just the people listed here)

Reply:
I really like your comment. You are right, you become a peer through merit, not by up-front nomination. Excellent!